features/old/accounts/personal_details.feature

```diff
@@ -35,12 +35,13 @@ Feature: Personal Details
     And I press "Update Details"
     Then I should be on the provider users page
 
-  @wip
   Scenario: Edit personal details with invalid data
-    And I go to the personal details page
-    And I fill in "Email" with ""
+    When I navigate to the Account Settings
+    And I go to the provider personal details page
+    And I fill in "Username" with ""
+    And I fill in "Current password" with "supersecret"
     And I press "Update Details"
-    Then I should see "can't be blank" within "#user_email_input"
+    Then I should see "is too short"  
```

Comment:
```suggestion
    Then I should see "is too short"
```

Reply:
Removing the trailing spaces.

It would actually be nice to add a step to validate where the error is shown.

So that the step would say `Then I should see "is too short" in inline errors for username input`